example/passthrough_hp.cc

```diff
@@ -43,7 +43,7 @@
  * \include passthrough_hp.cc
  */
 
-#define FUSE_USE_VERSION 35
+#define FUSE_USE_VERSION 311
```

Comment:
Let's instead use FUSE_MAKE_VERSION(3, 11) here. This will make sure that in the future, no one sets FUSE_USE_VERSION = 40 and gets confused because it sorts as less than 311.